src/components/MainMenu.jsx

```diff
@@ -1,18 +1,87 @@
 import styled from "styled-components";
 import { Routes, Route, NavLink, Link } from "react-router-dom";
+import logo from "../images/Logo_nacHHaltig.png";
+import NavLinks from "./NavLinks";
+
+import Restaurants from "../pages/Restaurants";
+import Shopping from "../pages/Shopping";
+import Beauty from "../pages/Beauty";
+import Favourites from "../pages/Favourites";
+import Calculator from "../pages/Calculator";
+import NotFound from "../pages/NotFound";
 
 function MainMenu() {
   return (
     <>
-      <nav>
-        <NavLink to="Restaurants">Restaurants</NavLink> <br />
-        <NavLink to="Shopping">Shopping</NavLink> <br />
-        <NavLink to="Beauty + Hair">Beauty + Hair</NavLink> <br />
-        <NavLink to="Favoriten">Favoriten</NavLink> <br />
-        <NavLink to="Co2-Rechner">Co2-Rechner</NavLink>
-      </nav>
+      <Header>
+        <Link to="/">
+          <LogoImage src={logo} alt="Logo" />
+        </Link>
+      </Header>
+
+      <Routes>
+        <Route
+          path="/"
+          element={
+            <>
+              <LogoImage />
+              <NavContainer>
+                <NavLinks />
+              </NavContainer>
+            </>
+          }
+        />
+        <Route path="restaurants" element={<Restaurants />}></Route>
+        <Route path="shopping" element={<Shopping />}></Route>
+        <Route path="beauty" element={<Beauty />}></Route>
+        <Route path="favoriten" element={<Favourites />}></Route>
+        <Route path="co2-rechner" element={<Calculator />}></Route>
+
+        <Route path="*" element={<NotFound />} />
+      </Routes>
     </>
   );
 }
 
 export default MainMenu;
+
+/* --- STYLING --- */
+
+const Header = styled.header`
+  background: var(--main-background);
+  border-bottom: 2px solid var(--logo-one);
+  text-align: center;
+`;
+
+const LogoImage = styled.img`
+  width: 20rem;
+  align-self: center;
+`;
+
+const NavContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 20%; // Elemente sollen sich gleichmäßig vertikal verteilen, mittig
```

Comment:
Is this comment necessary? If yes, please rewrite in English ;)

Reply:
I've deleted it, it's not necessary :)